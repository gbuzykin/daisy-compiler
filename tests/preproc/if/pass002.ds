#if true
#info "case #1"
#else
#info "case #2"
#endif

#if false
#info "case #3"
#else
#info "case #4"
#endif
