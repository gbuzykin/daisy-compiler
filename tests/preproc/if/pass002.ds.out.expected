./preproc/if/pass002.ds:2:2: info: "case #1"
 2 | #info "case #1"
   |  ^~~~
./preproc/if/pass002.ds:10:2: info: "case #4"
 10 | #info "case #4"
    |  ^~~~
./preproc/if/pass002.ds: info: warnings 0, errors 0
